cmd/jujud/agent/machine/manifolds.go

```diff
@@ -665,7 +666,7 @@ func commonManifolds(config ManifoldsConfig) dependency.Manifolds {
 		}),
 
 		charmhubHTTPClientName: dependency.Manifold{
-			Start: func(_ dependency.Context) (worker.Worker, error) {
+			Start: func(_ context.Context, getter dependency.Getter) (worker.Worker, error) {
```

Comment:
getter is just as unused here as the context, either both should be unnamed or both named.

Reply:
grep + sed didn't know what to do here in the original changes, I'll fix as many as I can.